cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -29,6 +30,17 @@
 ]
 
 
+def find_global_ipv6(vm):
+    """
+    Find global IPv6 on a VM if present and return it as a string. Otherwise, return None.
```

Comment:
Required:
This should be an `inline` comment.

Reply:
@digitronik  why do you say so, not sure that I agree?

If anything I would request that the `Return` values are properly specified in the `Return:` header of the docblock, as opposed to including them in the first line of the summary.

Reply:
Ahhh I miss return in doc string. Just checked format. I agreed with your statement Return block needs.